llava/eval/model_vqa.py

```diff
@@ -87,6 +91,7 @@ def eval_model(args):
     parser = argparse.ArgumentParser()
     parser.add_argument("--model-path", type=str, default="facebook/opt-350m")
```

Comment:
can you change line 92 and 93 based on https://github.com/nahidalam/LLaVA/blob/maya_eval/llava/eval/model_vqa_mmbench.py#L146-L147